google/cloud/spanner/value.h

```diff
@@ -194,10 +209,10 @@ class Value {
    * The type `T` may be any valid type shown above, except vectors of vectors
    * are not allowed.
    *
-   * NOTE: If `T` is a `std::tuple` with field names (i.e., at least one of its
-   * element types is a `std::pair<std::string, T>`) then, all of the vector's
-   * elements must have exactly the same field names. Any mismatch in in field
-   * names results in undefined behavior.
+   * @warning If `T` is a `std::tuple` with field names (i.e., at least one of
+   * its element types is a `std::pair<std::string, T>`) then, all of the
```

Comment:
Should we indent these?  We have done so in the other libraries, but I know it is tedious...

Reply:
Ditto for the `@note` paragraphs.

Reply:
SGTM. Indented wrapped lines 2 spaces